localsettings/services-parsoid.txt

```diff
@@ -0,0 +1,18 @@
+// Parsoid/PHP required configuration
+$wgEnableRestAPI = true;
```

Comment:
was going to suggest putting $wgEnableRestAPI in the overall core settings per above, but that configuration has been removed from core, so might not be needed at all

Reply:
Most of the Parsoid section is no longer needed for latest MediaWiki, but I wanted to keep it, because Patchdemo allows creating wikis using older MediaWiki versions too. See #82.